locksmith/__tests__/controllers/priceController.test.ts

```diff
@@ -1,55 +0,0 @@
-import request from 'supertest'
```

Comment:
Are we not using this controller anymore?

Reply:
It was skipped so far. We've not been running it at all for a while. 

The test needs a rewrite so I'm removing it. We already test the new price controller.

Reply:
works for me!

Reply:
Let's see if we can remove the old priceController completely?